### ISSUE TITLE
Executable's version is wrong

### ISSUE THREAD

**Issue by MCOfficer** [KoalaBear84/OpenDirectoryDownloader#50]
**Describe the bug**
Both `--version` and `--help` display the version as 1.0.0.

**To Reproduce**
1. Run `./OpenDirectoryDownloader --version`
2. Output:
```
❯ ./OpenDirectoryDownloader --version
tarted with PID 1539
OpenDirectoryDownloader 1.0.0
Error command line parameter 'VersionRequestedError'
Unhandled exception. System.NullReferenceException: Object reference not set to an instance of an object.
   at OpenDirectoryDownloader.Program.Main(String[] args)
   at OpenDirectoryDownloader.Program.<Main>(String[] args)
[1]    1539 abort      ./OpenDirectoryDownloader --version

```

**Expected behavior**
The version string should reflect the actual release version.

**Desktop (please complete the following information):**
 - OS: Ubuntu 18.04.5
 - Version: v1.9.1.0 self-contained, also happened with earlier versions.

**Comment by KoalaBear84** [comment]
Good catch. And I know why, because there is no version in code. It's currently only 'defined' by the Git Tag and nothing else.

But maybe there is a way to supply the version. 🙃😃

**Comment by KoalaBear84** [comment]
Fixed it with 9297bd0
